app/controllers/oidc/api_key_roles_controller.rb

```diff
@@ -113,19 +113,18 @@ def redirect_for_deleted
   PERMITTED_API_KEY_ROLE_PARAMS = [
     :name,
     :oidc_provider_id,
-    {
-      api_key_permissions: [:valid_for, { scopes: [], gems: [] }],
-      access_policy: {
-        statements_attributes: [
-          :effect,
-          { principal: :oidc, conditions_attributes: %i[operator claim value] }
-        ]
-      }
+    api_key_permissions: [:valid_for, scopes: [], gems: []],
+    access_policy: {
+      statements_attributes: [[
+        :effect,
+        principal: :oidc,
+        conditions_attributes: [%i[operator claim value]]
```

Comment:
Hey @segiddins this line doesn't fail any tests either way, but it looks like this is how the code works. I guess we don't pass conditions in a test?

Reply:
It's possible